lib/src/configuration.cc

```diff
@@ -861,16 +861,16 @@ void Configuration::parseConfigFile()
             // broker-ws-uris is restricted to the config file, so it's handled
             // specially here. It's not part of defaults_ because all the extra
             // meta-data for Horsewhisperer isn't needed.
-            // TODO: add `servers`, which will populate broker-ws-uris and master-uris
+            // TODO: add `servers`, which will populate broker-ws-uris and primary-uris
             //       using default ports; expects hostnames only
             if (key == "broker-ws-uris") {
                 wrap(key, lth_loc::translate("array of strings"), [&]() {
                     broker_ws_uris_ = config->get_string_list(key);
                 });
                 continue;
-            } else if (key == "master-uris") {
+            } else if (key == "master-uris" || key == "primary-uris") {
```

Comment:
It looks like whichever one is parsed last will win here. I think we should try to prefer `primary-uris` if they're both set.